packages/@react-aria/sidenav/src/useSideNavItem.ts

```diff
@@ -36,7 +36,8 @@ export function useSideNavItem<T>(props: SpectrumSideNavItemProps<T>, state: Tre
 
   let isDisabled = state.disabledKeys.has(item.key);
   let isSelected = state.selectionManager.isSelected(item.key);
-  let {pressProps} = usePress({...itemProps, isDisabled});
+  let {onPressStart, onPress, ...otherProps} = itemProps;
```

Comment:
same thing here

Reply:
Should some of our hooks also have things like
```
  let [usePressProps, remainingProps] = separatePressProps(props);
  let {pressProps} = usePress({...usePressProps, isDisabled});
```

Reply:
actually, why do we do `...mergeProps(itemProps, pressProps)` in listItemProps in the first place? We pass the entirety of `itemProps` to `usePress` which should return everything in `itemProps` merged with the `pressProps`   via ` pressProps: mergeProps(domProps, pressProps)`?

Reply:
Right, pressProps should already do the merging internally, and take out the event handlers, so we shouldn't need to merge with otherProps